public/js/dom.js

```diff
@@ -0,0 +1,44 @@
+// const url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query=`;
+function selecElement(id) {
+	return document.querySelector(id);
+}
+function addEventListenerForSelector(selectorElement, event, callback){
+	return document.querySelector(selectorElement).addEventListener(event,callback);
+}
+function createElement(e){
+	return document.createElement(e);
+}
+addEventListenerForSelector('#submit-btn', 'submit', (event) => {
+	event.preventDefault();
+	if (selecElement('#searchInput').value == ""){
+		alert('Entar a movie name');
```

Comment:
don't use ``` alert ``` again 
alert for the developer not for the user 
instead of ``` alert ```  you can add ``` span ``` under the input and   that span be empty at first time when the user enter something wrong fill it with the error message


https://stackoverflow.com/questions/20359807/why-shouldnt-i-use-alert-in-javascript